project/Dependencies.scala

```diff
@@ -3,102 +3,125 @@ import sbt._
-  val googleV       = "1.23.0"
+  val akkaV = "2.5.22"
+  val akkaHttpV = "10.1.8"
+  val jacksonV = "2.9.9"
+  val jacksonDatabindV = "2.9.9.2" // jackson-databind has a security patch on the 2.9.9 branch
+  val googleV = "1.23.0"
   val automationGoogleV = "1.30.5"
   val scalaLoggingV = "3.9.0"
-  val scalaTestV    = "3.0.8"
-  val slickV        = "3.2.3"
+  val scalaTestV = "3.0.8"
+  val slickV = "3.2.3"
   val http4sVersion = "0.21.0-M5" //remove http4s related dependencies once workbench-libs are upgraded
-  val guavaV        = "28.0-jre"
+  val guavaV = "28.0-jre"
 
-  val workbenchUtilV    = "0.5-4c7acd5"
-  val workbenchModelV   = "0.13-6dc016b"
-  val workbenchGoogleV  = "0.21-58c913d"
-  val workbenchGoogle2V = "0.6-f846fad"
+  val workbenchUtilV = "0.5-4c7acd5"
+  val workbenchModelV = "0.13-6dc016b"
+  val workbenchGoogleV = "0.21-890a74f"
+  val workbenchGoogle2V = "0.6-8bae8e8"
   val workbenchMetricsV = "0.3-c5b80d2"
   val workbenchNewRelicV = "0.3-8bae8e8"
 
-  val excludeAkkaActor          = ExclusionRule(organization = "com.typesafe.akka", name = "akka-actor_2.12")
-  val excludeGuavaJDK5          = ExclusionRule(organization = "com.google.guava", name = "guava-jdk5")
-  val excludeGuava              = ExclusionRule(organization = "com.google.guava", name = "guava")
-  val excludeWorkbenchUtil      = ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = "workbench-util_2.12")
-  val excludeWorkbenchModel     = ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = "workbench-model_2.12")
-  val excludeWorkbenchMetrics   = ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = "workbench-metrics_2.12")
-  val excludeIoGrpc             = ExclusionRule(organization = "io.grpc", name = "grpc-core")
-  val excludeFindbugsJsr        = ExclusionRule(organization = "com.google.code.findbugs", name = "jsr305")
-  val excludeGson               = ExclusionRule(organization = "com.google.code.gson", name = "gson")
-  val excludeGoogleApiClient    = ExclusionRule(organization = "com.google.api-client", name = "google-api-client")
-  val excludeGoogleApiClientJackson2  = ExclusionRule(organization = "com.google.http-client", name = "google-http-client-jackson2")
-  val excludeGoogleHttpClient   = ExclusionRule(organization = "com.google.http-client", name = "google-http-client")
-  val excludeJacksonCore        = ExclusionRule(organization = "com.fasterxml.jackson.core", name =  "jackson-core")
-  val excludeJacksonAnnotation  = ExclusionRule(organization = "com.fasterxml.jackson.core", name = "jackson-annotations")
-  val excludeSlf4j              = ExclusionRule(organization = "org.slf4j", name = "slf4j-api")
-  val excludeLogbackCore        = ExclusionRule(organization = "ch.qos.logback", name = "logback-core")
+  val excludeAkkaActor = ExclusionRule(organization = "com.typesafe.akka", name = "akka-actor_2.12")
```

Comment:
Oof.. should we avoid formatting `Dependencies.scala`?

Reply:
I hear you. I manually reverted most, if not all, of them and excluded this file in scalafmt config.

Reply:
Still a lot of changes in the diff but I can live with it if it's just indentation.